monitoring/generator.go

```diff
@@ -189,8 +189,8 @@ type Observable struct {
 	//
 	PossibleSolutions string
 
-	// PanelOptions describes some options for how to render the metric in the Grafana panel.
-	PanelOptions panelOptions
+	// NewPanelOptions describes some options for how to render the metric in the Grafana panel.
```

Comment:
This looks wrong, it no longer starts with the field name. I am surprised the linter didn't complain about this at all.

```suggestion
	// PanelOptions describes some options for how to render the metric in the Grafana panel.
```